math/ruy/CMakeLists.txt

```diff
@@ -44,17 +44,21 @@ if(CONFIG_MATH_RUY)
     endif()
   endif()
 
+  # cmake-format: off
+
   # ############################################################################
   # Include Directory
   # ############################################################################
 
-  set(INCDIR ${CMAKE_CURRENT_LIST_DIR}/ruy)
+  # set(INCDIR ${CMAKE_CURRENT_LIST_DIR}/ruy)
```

Comment:
why comment out

Reply:
but why not fix it?

Reply:
Many libraries depend on ruy and only need header file support, but tflm does not need it. Why spend time compiling it?and also, this is a issue submitted by the original author, and I have no interest in support it anymore

Reply:
if so, just remove the code instead comment out the unworked code.

Reply:
No, but the header file of ruy is working